safehttp/handler.go

```diff
@@ -14,12 +14,25 @@
 
 package safehttp
 
-// Handler TODO
+// Handler responds to an HTTP request.
+//
+// ServeHTTP should set headers, call one of the writing functions on the ResponseWriter
+// to write the body of the response and then return the Result received from writing
+// to the ResponseWriter.
```

Comment:
It doesn't have to set the headers, so leave it out.

Instead of technically saying "should call one of the writing functions on the Response writer to write the body of the response", it's enough to just say "write the response exactly once, returning the result".